services/user.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
The import of `User` from `db.models` is unnecessary as `get_user_model()` is used for user operations. Consider removing this import.